docs/Tutorials/OpenAPI.md

```diff
@@ -488,18 +673,226 @@ As JSON, this could look as follows:
+        New-PodeOAObjectProperty  -Name 'User' -XmlName 'user'  |
+        Add-PodeOAComponentSchema
+
+    New-PodeOAStringProperty -Name 'street' -Example '437 Lytton' -Required |
+        New-PodeOAStringProperty -Name 'city' -Example 'Palo Alto' -Required |
+        New-PodeOAStringProperty -Name 'state' -Example 'CA' -Required |
+        New-PodeOAStringProperty -Name 'zip' -Example '94031' -Required |
+        New-PodeOAObjectProperty -Name 'Address' -XmlName 'address' -Description 'Shipping Address' |
+        Add-PodeOAComponentSchema
+
+    Merge-PodeOAProperty -Type AllOf -ObjectDefinitions 'Address','User'
+
+```
+
+As JSON, this could look as follows:
+
+```json
+{
+  "allOf": [
+    {
+      "$ref": "#/components/schemas/Address"
+    },
+    {
+      "$ref": "#/components/schemas/User"
+    }
+  ]
+}
+```
+
+
+## OpenApi Documentation pages
 
 If you're not using a custom OpenAPI viewer, then you can use one of the inbuilt ones with Pode - either Swagger and ReDoc, or both!
 
 For both you can customise the path to access the page on, but by default Swagger is at `/swagger` and ReDoc is at `/redoc`. If you've written your own custom OpenAPI definition then you can also set a custom path to fetch the definition.
 
-To enable either you can use the [`Enable-PodeOpenApiViewer`](../../Functions/OpenApi/Enable-PodeOpenApiViewer) function:
+To enable either you can use the [`Enable-PodeOAViewer`](../../Functions/OpenApi/Enable-PodeOAViewer) function:
 
 ```powershell
 # for swagger at "/docs/swagger"
 Enable-PodeOpenApiViewer -Type Swagger -Path '/docs/swagger' -DarkMode
 
-# or ReDoc at the default "/redoc"
+# and ReDoc at the default "/redoc"
 Enable-PodeOpenApiViewer -Type ReDoc
+
+# and RapiDoc at "/docs/rapidoc"
+Enable-PodeOAViewer -Type RapiDoc -Path '/docs/rapidoc' -DarkMode
+
+# and StopLight at "/docs/stoplight"
+Enable-PodeOAViewer -Type StopLight -Path '/docs/stoplight'
+
+# and Explorer at "/docs/explorer"
+Enable-PodeOAViewer -Type Explorer -Path '/docs/explorer'
+
+# and RapiPdf at "/docs/rapipdf"
+Enable-PodeOAViewer -Type RapiPdf -Path '/docs/rapipdf'
+
+# plus a bookmark page with the link to all documentation
+Enable-PodeOAViewer -Type Bookmarks -Path '/docs'
+```
+## Multiple OpenAPI definition
+Is possible to have multiple OpenAPI definitions inside the same Server instance.
+This feature can be very usefull in situatione like:
+- Multiple versions of the OpenAPI specification for different use case
+- Same OpenAPI but with one using the OpenAPI spec 3.0.3 and another using 3.1.0
+- Different APIs based on the IP or URL
+
+### How to use it
+Any Pode funtion that interact with OpenAPI as a `-DefinitionTag` [string[]] parameter. Here you can be specify what definition you are going to interact.
+
+Note These function accept a simple string not an array
+- `Get-PodeOADefinition`
+- `Enable-PodeOpenApi`
+- `Enable-PodeOAViewer`
+- `Add-PodeOAInfo`
+- `Test-PodeOAJsonSchemaCompliance`
```

Comment:
Tweak to be:

```markdown
Any Pode function that interacts with OpenAPI has a `-DefinitionTag` `[string[]]` parameter available. This allows you to specify within which OpenAPI definition(s) the API's definition should be available.

!!! note
    These functions accept a simple string, and not an array

    * `Get-PodeOADefinition`
    * `Enable-PodeOpenApi`
    * `Enable-PodeOAViewer`
    * `Add-PodeOAInfo`
    * `Test-PodeOAJsonSchemaCompliance`
```

`!!! note` has a special rendering in mkdocs:
![image](https://github.com/Badgerati/Pode/assets/1483395/48a7abcf-a27a-4ffd-83a1-2e5bb97d9591)